src/data/equiv/basic.lean

```diff
@@ -495,6 +495,12 @@ def subtype_congr {p : α → Prop} {q : β → Prop}
  λ ⟨x, h⟩, subtype.eq' $ by simp,
  λ ⟨y, h⟩, subtype.eq' $ by simp⟩
 
+def subtype_congr_right {p q : α → Prop} (e : ∀x, p x ↔ q x) : subtype p ≃ subtype q :=
```

Comment:
This is actually an equality, no? Is there a reason to prefer it stated like this?

Reply:
I guess... There are much more facts about equivs than equalities, so equivs are probably easier. Maybe it's too similar to `subtype_congr_prop`, but that has a less useful condition (you're unlikely to prove `p = q` without going via `∀x, p x ↔ q x`)

Reply:
Hmm, yeah, I had something in mind when I wrote that but now I'm not sure what. It would be a non-defeq equality between types. Probably better to keep it as an equiv.